lib/cisco_node_utils/cmd_ref/radius_server.yaml

```diff
@@ -30,35 +38,70 @@ authentication:
   set_value: '<state> radius-server host <ip> authentication'
   default_value: true
 
+host_port_pairs:
+  _exclude: [nexus]
+  multiple: true
+  get_command: "show running-config radius-server"
+  get_value: '/^radius-server host <ip> auth-port (\d+) acct-port (\d+)/'
+  set_value: '<state> radius-server host <ip> auth-port <auth_port> acct-port <acct_port>'
```

Comment:
It seems like this should be the set_value for `hosts` on IOS XR (having `host_port_pairs` be a getter-only property)